Mathlib/Analysis/SpecialFunctions/ContinuousFunctionalCalculus/Rpow/Basic.lean

```diff
@@ -707,10 +719,12 @@ theorem _root_.CStarAlgebra.isStrictlyPositive_TFAE {a : A} :
      ∃ b, IsUnit b ∧ IsSelfAdjoint b ∧ a = b * b,
      ∃ b, IsUnit b ∧ a = star b * b,
      ∃ b, IsUnit b ∧ a = b * star b,
+     IsUnit a⁺ ∧ a = a⁺,
+     IsUnit a ∧ IsSelfAdjoint a ∧ a⁻ = 0,
```

Comment:
I'm not convinced about these spellings, because they are just a `rw` away from the `0 ≤ a ∧ IsUnit a` one below. Note that these are different from the ones above because the ones above assume that `IsUnit b` instead, not `IsUnit a`.